lib/tapioca/internal.rb

```diff
@@ -16,6 +16,18 @@
 require "tapioca/executor"
 require "tapioca/symbol_table_parser"
 require "tapioca/symbol_loader"
+require "tapioca/compilers/gem/events"
+require "tapioca/compilers/gem/listeners/base"
+require "tapioca/compilers/gem/listeners/dynamic_mixins"
+require "tapioca/compilers/gem/listeners/methods"
+require "tapioca/compilers/gem/listeners/mixins"
+require "tapioca/compilers/gem/listeners/sorbet_enums"
+require "tapioca/compilers/gem/listeners/sorbet_helpers"
+require "tapioca/compilers/gem/listeners/sorbet_props"
+require "tapioca/compilers/gem/listeners/sorbet_required_ancestors"
+require "tapioca/compilers/gem/listeners/sorbet_signatures"
+require "tapioca/compilers/gem/listeners/sorbet_type_variables"
+require "tapioca/compilers/gem/listeners/yard_doc"
```

Comment:
Can we move these requires into `tapioca/compilers/gem/listeners.rb` and just require that from here?

Reply:
Done in [7472541](https://github.com/Shopify/tapioca/pull/785/commits/747254144a45900143805df4abe1cc1ef4b22d72) 👍